forge-gui/res/cardsfolder/a/armageddon_clock.txt

```diff
@@ -8,4 +8,5 @@ SVar:TrigDealDamage:DB$ DealDamage | Defined$ Player | NumDmg$ X
 SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ DOOM | CounterNum$ 1
 SVar:X:Count$CardCounters.DOOM
 AI:RemoveDeck:All
+DeckHas:Ability$Counters
```

Comment:
not sure this needs to be added to a RemoveDeck:All card

Reply:
These would still get added to random deck of the player no ? (not sure though)